evalml/pipelines/time_series_pipeline_base.py

```diff
@@ -42,6 +42,12 @@ def __init__(
         self.max_delay = self.pipeline_params["max_delay"]
         self.forecast_horizon = self.pipeline_params["forecast_horizon"]
         self.time_index = self.pipeline_params["time_index"]
+        self.series_id = (
+            self.pipeline_params["series_id"]
+            if "series_id" in self.pipeline_params
+            else None
+        )
+        self.is_multiseries = False if self.series_id is None else True
```

Comment:
Why are these necessary for this issue? I handle this in the multiseries pipeline object

Reply:
Leftover from when I thought we're using the existing base, will revert.